src/cookbook/navigation/set-up-universal-links.md

```diff
@@ -0,0 +1,202 @@
+</noscript>
+
+At this point, the application side is ready to go.
+
+## 3. Hosting apple-app-site-association file
+
+You need to host an apple-app-site-association file in the web domain.
+This file tells the mobile browser which iOS application it should open
+instead of opening in the browser. You need the app ID of the Flutter
+application created in the previous step in order to create the file.
+
+### App ID
+
+The app ID has the format of \<team id\>.\<bundle id\>. The bundle ID
+is found in the Xcode project, and the team ID is found in
+[developer account][]. If the team ID is <strong>S8QB4VV633</strong>
+and bundle ID is <strong>com.myapp.universallinks</strong>, the app ID
+is then <strong>S8QB4VV633.com.myapp.universallinks</strong>.
+
+### apple-app-site-association
+
+The hosted file should have the following content:
+```json
+{
+  "applinks": {
+      "details": [
+      {
+        "appID": "S8QB4VV633.com.myapp.universallinks",
+        "paths": ["*"]
+      }
+    ]
+  }
+}
+```
+
+Replace the value of <strong>appID</strong> with your own. The
+<strong>paths</strong> field specifies the allowed universal links,
+and using a <strong>*</strong> means every path is redirected
+to the Flutter application. Adjust the paths as desired.
+
+Host the file at the following URL:
+<strong>\<webdomain\>/.well-known/apple-app-site-association</strong>.
+If using GitHub Pages, your repository should look similar to this
+[sample repo]. Once this is done, make sure that the file is
+accessible directly from your browser.
+
+
+## Testing
+{{site.alert.note}}
+It might take up to 24 hours before Apple’s CDN requests the AASA
+file from your web domain. The universal link won’t work until it
+does. To bypass Apple’s CDN, see the [alternate mode section].
+{{site.alert.end}}
+
+You can use a real device or the Simulator to test the universal link.
+Before you actually test the universal links, make sure you have
+executed <strong>flutter run</strong> at least once on the device. This ensures that the
+Flutter application is installed.
+
+<noscript>
+  <img src="/assets/images/docs/cookbook/set-up-universal-links-simulator.png" alt="Simulator screenshot"/>
+</noscript>
+
+If using the Simulator, you can test using Xcode CLI.
+```shell
+xcrun simctl openurl booted https://<web domain>/details
+```
+
+Otherwise, type the url in the Note app and click it.
+
+If everything is set up correctly, the Flutter application
+launches and displays the details screen:
+
+<noscript>
+  <img src="/assets/images/docs/cookbook/set-up-universal-links-simulator-deeplinked.png" alt="Simulator screenshot"/>
+</noscript>
+
+[alternate mode section]: https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_developer_associated-domains?language=objc
+[sample repo]: https://github.com/chunhtai/chunhtai.github.io
```

Comment:
A note to myself, need to replace this with more appropriate github page account